template/iconComponent.ts

```diff
@@ -0,0 +1,38 @@
+interface TemplateProps {
+    fileName: string;
+    data: string;
+}
+
+function componentTemplate ({fileName,data}:TemplateProps) {
+    const render = `
+// GENERATED BY ./scripts/iconScript.ts
+
+import * as React from 'react';
+import { IconProps } from '../helper';
+
+function Eos${fileName}({size = "m", color = "black", rotate = 0}: IconProps) {
+    const valuesMap: any = {
+        xs: 4,
+        s: 8,
+        base: 14,
+        m: 18,
+        l: 24,
+        xl: 32,
+        xxl: 48,
+        xxxl: 64,
+    }
+    let sizes: any = ["xs","s","base","m","l","xl","xxl","xxxl"]
```

Comment:
Maybe its a good idea to export those as part of some `utils/constants.ts` file where we could have all the constants needed, ex
```js
export default {
   SIZES: ["xs","s","base","m","l","xl","xxl","xxxl"]
}

or have a look at how we do it here: https://github.com/EOS-uiux-Solutions/eos-icons-api/blob/main/src/common/constants.ts

This way we can have all constants in the same place and we can lately import them when needed

Reply:
yes, you can export this since it might be used at more places. also, if you are using this array just to check if passed size is present in it or not, you can do something like `Object.keys(valuesMap).includes(size)`. This way we dont need the array.
Reason for that is, if we add more sizes, we'll have more places to update it. It's good if we can reuse the existing code.

Reply:
Thanks @en3sis @kbhutani0001, I was using that array for checking if passed size is present or not. I have updated it to what @kbhutani0001 suggested. Now we only have one place where the sizes are being defined.

Reply:
Ah yes sorry, on my example I'm showing the sizes as an Array but I was talking about the object itself :P
Good catch about checking the keys  @kbhutani0001

Reply:
I have added the `valuesMap` object as a export in helper.ts file